src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,164 @@
+  <string name="proxy_port">Proxy-Port</string>
+  <string name="proxy_type">Proxy-Typ</string>
+  <string name="repositories">Paketquellen</string>
+  <string name="repository">Paketquelle</string>
+  <string name="repository_not_used_DESC">Diese Paketquelle wurde noch nicht verwendet. Bitte Aktivieren um die angebotenen Apps anzusehen.</string>
+  <string name="repository_unsigned_DESC">Unsigniert. App-Liste konnte nicht verifiziert werden. Vorsicht bei Downloads aus unsignierten Paketquellen.</string>
+  <string name="requires_FORMAT">Benötigt %s</string>
+  <string name="save">Speichern</string>
+  <string name="saving_details">Speichere Details</string>
+  <string name="screenshots">Screenshots</string>
+  <string name="search">Suche</string>
+  <string name="select_mirror">Wähle einen Mirror</string>
+  <string name="show_more">Zeige mehr</string>
+  <string name="show_older_versions">Zeige ältere Versionen</string>
+  <string name="signature_FORMAT">Signatur %s</string>
+  <string name="signed_using_unsafe_algorithm">Signiert mit unsicherem Algorithmus</string>
+  <string name="skip">Überspringen</string>
+  <string name="socks_proxy">Socks-Protokoll</string>
+  <string name="sorting_order">Sortierreihenfolge</string>
+  <string name="source_code">Quellcode</string>
+  <string name="source_code_no_longer_available">Quellcode nicht mehr verfügbar</string>
+  <string name="suggested">Empfohlen</string>
+  <string name="sync_repositories">Synchronisiere Paketquellen</string>
+  <string name="sync_repositories_automatically">Paketquellen automatisch synchronisieren</string>
+  <string name="syncing">Synchronisiere</string>
+  <string name="syncing_FORMAT">Synchronisiere %s</string>
+  <string name="system">System</string>
+  <string name="tap_to_install_DESC">Tippen zum Installieren.</string>
+  <string name="theme">Thema</string>
+  <string name="tracks_or_reports_your_activity">Verfolgt oder meldet Aktivitäten</string>
+  <string name="uninstall">Deinstallieren</string>
+  <string name="unknown">Unbekannt</string>
+  <string name="unknown_error_DESC">Unbekannter Fehler.</string>
+  <string name="unknown_FORMAT">Unbekannt: %s</string>
+  <string name="unsigned">Unsigniert</string>
+  <string name="unstable_updates">Instabile Updates</string>
+  <string name="unstable_updates_summary">Installation instabiler Versionen vorschlagen</string>
+  <string name="unverified">Ungeprüft</string>
+  <string name="update">Update</string>
+  <string name="updates">Updates</string>
+  <string name="upstream_source_code_is_not_free">Upstream-Quellcode ist nicht frei</string>
+  <string name="username">Benutzername</string>
+  <string name="username_missing">Benutzername fehlt</string>
+  <string name="validation_index_error_DESC">Index konnte nicht validiert werden.</string>
+  <string name="version_FORMAT">Version %s</string>
+  <string name="versions">Versionen</string>
+  <string name="waiting_to_start_download">Warte auf Download-Start</string>
```

Comment:
```suggestion
  <string name="waiting_to_start_download">Warte auf Herunterladen-Start</string>
```

Reply:
Looks and sounds stupid. Would then change this to:

Warte auf Herunterladevorgang